stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamShapes.kt

```diff
@@ -13,21 +13,24 @@ import androidx.compose.ui.unit.dp
  * @param otherMessageBubble - The bubble that wraps other people's message content.
  * @param inputField - The shape of the input field.
  * @param attachment - The shape of attachments.
+ * @param imageThumbnail - The shape of image thumbnails, shown in selected attachments and image file attachments.
  * */
 public class StreamShapes(
     public val avatar: Shape,
     public val myMessageBubble: Shape,
     public val otherMessageBubble: Shape,
     public val inputField: Shape,
-    public val attachment: Shape
+    public val attachment: Shape,
+    public val imageThumbnail: Shape
 ) {
     public companion object {
         public val default: StreamShapes = StreamShapes(
             avatar = CircleShape,
             myMessageBubble = RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp, bottomStart = 16.dp),
             otherMessageBubble = RoundedCornerShape(topStart = 16.dp, topEnd = 16.dp, bottomEnd = 16.dp),
             inputField = RoundedCornerShape(24.dp),
-            attachment = RoundedCornerShape(16.dp)
+            attachment = RoundedCornerShape(16.dp),
+            imageThumbnail = RoundedCornerShape(8.dp)
```

Comment:
Lack of trailing commas in a bunch of places on this PR - it's why this file has more line changes than it needs to, they were already missing at the ends of both lists that are modified here.

We're awaiting the next release of ktlint that will hopefully place trailing commas more aggressively than the Studio formatter.

Reply:
Yeah it's been inconsistent for me and something I easily forget (not used to putting them honestly). I'll go through some of these files and add them where I see them missing